shardingsphere-mode/shardingsphere-mode-type/shardingsphere-cluster-mode/shardingsphere-cluster-mode-core/src/main/java/org/apache/shardingsphere/mode/manager/cluster/ClusterContextManagerBuilder.java

```diff
@@ -150,10 +152,12 @@ private ContextManager createContextManager(final ClusterPersistRepository repos
                                                 final Properties transactionProps, final ModeConfiguration modeConfiguration) {
         InstanceContext instanceContext = new InstanceContext(metaDataPersistService.getComputeNodePersistService().loadComputeNodeInstance(instanceDefinition),
                 new ClusterWorkerIdGenerator(repository, metaDataPersistService, instanceDefinition), modeConfiguration);
+        DistributeLockContext distributeLockContext = new DistributeLockContext(instanceContext, new GlobalLockRegistryService(repository));
+        distributeLockContext.synchronizeGlobalLock();
```

Comment:
what `synchronizeGlobalLock` will do here?

Reply:
Three things to do here
1. Synchronize the global lock state,
2. Lock resources,
3. ACK response